server/src/test/java/org/elasticsearch/index/mapper/CompletionFieldTypeTests.java

```diff
@@ -1,59 +0,0 @@
-/*
- * Licensed to Elasticsearch under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.elasticsearch.index.mapper;
-
-import org.elasticsearch.search.suggest.completion.context.ContextBuilder;
-import org.elasticsearch.search.suggest.completion.context.ContextMappings;
-import org.junit.Before;
-
-import java.util.Arrays;
-
-public class CompletionFieldTypeTests extends FieldTypeTestCase {
```

Comment:
Should this have been deleted?

Reply:
Again, none of these values can be updated in a merge, so we don't need an equality check.